app/views/smart_answers/question.html.erb

```diff
@@ -19,19 +19,13 @@
 
           <% if question.body.present? && show_body %>
             <article role="article">
-              <%= render "govuk_publishing_components/components/govspeak", {
-              } do %>
-                <%= question.body %>
-              <% end %>
```

Comment:
Yay this looks great 🥇 Wonder why they all have an empty object spanning two lines 🤔